cfa_azure/helpers.py

```diff
@@ -934,8 +934,8 @@ def add_task_to_job(
         )
         batch_client.task.add(job_id=job_id, task=task)
         logger.debug(f"Task '{task_id}' added to job '{job_id}'.")
-        return task_id
-
+        tasks.append(task_id)
+    return tasks

```

Comment:
this is still returning a list. We should just return task_id from this function and there's no need to append it to tasks. Check out the master branch for how it should work